src/plays/markdown-editor/MarkdownEditor.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import PlayHeader from 'common/playlists/PlayHeader';
+import { Remarkable } from 'remarkable';
+import './styles.css';
+
+
+const md = new Remarkable();
+
+// WARNING: Do not change the entry component name
+function MarkdownEditor(props) {
+
+  // Your Code Start below.
+
+   // state for textarea input value
+   const [ text, setText ] = useState("");
+
+  return (
+    <>
+      <div className="play-details">
```

Comment:
Add a unique class name here along with `play-details`. Like 

```html
<div className="eboh-md-editor play-details">
```

Now use this one as a prefix to the styles in your CSS file. Like
```css
.eboh-md-editor.play-details {


} 

.eboh-md-editor .wrapper-div {

}
// Like the above.
```